elixir_grpc_bench/config/dev.exs

```diff
@@ -1,9 +1,9 @@
 import Config
 
-config :falco, start_server: true
+config :grpc, start_server: true
 
 config :logger,
-  level: :error,
-  compile_time_purge_level: :error,
+  level: :debug,
```

Comment:
won't setting logger to debug hamper the performance? or it doesn't matter here?

Reply:
Hi @LesnyRumcajs. We do not issue messages on the service so it has no effect.

Reply:
alright, sounds good!